src/main/resources/default/templates/wondergem/scripts/tokenfield.html.pasta

```diff
@@ -61,9 +61,6 @@
                     input.updateTokenRemovalVisibility();
 
                     this.element.on('tokenfield:removedtoken', function (e) {
-                        var identifier = e.attrs.value;
-                        $('input[data-identifier="' + identifier + '"]').remove();
-
```

Comment:
i can't see why we do this, but I may miss something

Reply:
I believe this were hidden inputs which where added to be available in posts. Theses were probably removed again.

I reimplemented the hidden inputs in https://github.com/scireum/sirius-web/pull/540/commits/d3acd32c5ca7e757901b18c26a1907b571a3035d